src/main/java/bio/terra/service/dataset/DatasetDao.java

```diff
@@ -660,6 +682,12 @@ public DatasetSummary mapRow(ResultSet rs, int rowNum) throws SQLException {
         throw new CorruptMetadataException(
             String.format("Invalid billing profiles for dataset - id: %s", datasetId), e);
       }
+      boolean isAzure =
+          storageResources.stream()
+              .anyMatch(sr -> CloudPlatformWrapper.of(sr.getCloudPlatform()).isAzure());
```

Comment:
It seems odd to create a wrapper to only call `isAzure()`. Is this different from
```suggestion
              .anyMatch(sr ->sr.getCloudPlatform() == CloudPlatform.AZURE);
```
?